src/Hazelcast.Net.Tests/Sandbox/SupportTests.cs.not

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
```

Comment:
What's this file about?

Reply:
Should not be part of the PR, can be removed.